components/molecules/LogoAndNameMobile.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { IoIosRose } from 'react-icons/io';
+
+import { Grey010 } from 'components/styling/colors';
+import { DefaultParagraph } from 'components/atoms/typography.styles';
+
+const MobileLeftDiv = styled.div`
+    background-color: ${Grey010};
+    display: grid;
+    grid-template-columns: 36px 1fr; 
+    cursor: pointer;
+`;
+
+const LogoDiv = styled.div`
```

Comment:
I'd create folders for the components, and separate these styled component definitions, to declutter.

Reply:
We have been using a system similar to that, but kept files like this when they were still short this way. To me personally, it is more comfortable to have few-lines-long code snippets at the place where they are used if the file remains readable. Makes it easier to debug for me. 

I agree that creating folders is something that will need to be introduced pretty soon to keep it organized. (Or subfolders within atoms, molecules etc.. Anyway, atomic design is something I am just trying out right now, which doesn't mean I want to stick to it. I mean I like it so far, but I think I can be persuaded to leave it, at least until I am not too invested in it :) Needless to say, I will not make up my mind without good reasoning :))

Reply:
If it makes you more comfortable leave it, I *personally* don't like seeing the css in my component 😄 .